malwarefront/src/components/Relations.js

```diff
@@ -1,9 +1,37 @@
 import React, {Component} from 'react';
+import {Link} from 'react-router-dom';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
 import {TagList} from './Tag';
-
+import api from "@malwarefront/api";
 import { ObjectLink } from '@malwarefront/ui';
+import RelationsAddModal from "./RelationsAddModal";
 
 class RelationsBox extends Component {
+    state = {
+        isAttributeAddModalOpen: false,
+        modalError: ""
+    }
+
+    addObjectRelations = async (relation, value) => {
+        try {
+            if (relation === "child")
+                await api.addObjectRelation("object", this.props.id, value)
+            else if (relation === "parent")
+                await api.addObjectRelation("object", value, this.props.id)
```

Comment:
According to `type` argument removal
```suggestion
            if (relation === "child")
                await api.addObjectRelation(this.props.id, value)
            else if (relation === "parent")
                await api.addObjectRelation(value, this.props.id)
```